dadi/lib/handlers/image.js

```diff
@@ -97,6 +97,10 @@ const ImageHandler = function (format, req, {
   this.exifData = {}
   this.isExternalUrl = !pathname.indexOf('http://') || !pathname.indexOf('https://')
 
+  this.noCache = this.parsedUrl.original.query &&
```

Comment:
Can we name this something a bit more descriptive, like `this.shouldBypassCache`?

Reply:
And yes.